src/BiscuitDisplayer.php

```diff
@@ -1,8 +1,22 @@
 <?php
 
 namespace BiscuitApp;
+
 require_once 'vendor/autoload.php';
+
 class BiscuitDisplayer
 {
+    static public function printBiscuits(array $allBiscuitObjects): string
+    {
+        $biscuitsString = '';
+        foreach ($allBiscuitObjects as $biscuitObject) {
+            $name = $biscuitObject->getName();
+            $image = $biscuitObject->getImg();
+            $RDT = $biscuitObject->getRDT();
 
-}
\ No newline at end of file
+            $biscuitsString .=
+                '<div class="card"><div class="item-header"><h2>' . $name . '</h2></div><div class="item-img"><img src="' . $image . '" alt="' . $name . ' biscuit' . '"></div><div class="item-data"><h3>Recommended Dunking Time:</h3><h3>' . $RDT . ' Seconds</h3></div></div>';
+        }
```

Comment:
capitalisation not the same as in the picture I provided - should be:

Recommended dunking time:
6 seconds

also, having two separate <h3> tags underneath each other is not quite correct use of h3, so you should change the second <h3> to be a <p> instead.